frontend/public/components/_nav.scss

```diff
@@ -52,27 +53,28 @@ $active-border: 3px;
   -webkit-overflow-scrolling: touch;
 }
 
+.navigation-container--open {
+  background-color: $color-pf-black-800;
+}
+
 .navigation-container__list {
   margin: -10px 0 10px 0;
   overflow-y: hidden;
   padding-left: 0;
   transition: max-height 0.25s;
 }
 
-.navigation-container__section, .navigation-container__section__separator {
+.navigation-container__section {
   border-bottom: 1px solid $color-os-nav-item-seperator;
 }
 
+.navigation-container__section__separator {
+  border-bottom: 1px solid $color-pf-black-900;
+}
+
 .navigation-container__section--active {
-  background-color: $color-os-nav-background-active;
+  background-color: $color-pf-black-700;
   position: relative;
-  &:before {
```

Comment:
We currently don't have nav items without children.  So either this entire block (lines 66-76) and [supporting jsx](https://github.com/openshift/console/blob/master/frontend/public/components/nav.jsx#L225) should go or we should keep the entire rule.